model/bin/w3_make

```diff
@@ -171,6 +171,7 @@
   # MULTI MULTI_ESMF SBS1
   if [ -n "`grep SCRIPNC $switch_file`" ] || [ -n "`grep OASIS $switch_file`" ]  || [ -n "`grep PDLIB $switch_file`" ]
   then
+    cdf_programs="$cdf_programs ww3_grid"
```

Comment:
@aliabdolali  ww3_grid should still be removed from line 138 and then  added to the regular programs in the else statement for reg programs in this list.

Reply:
Done